### DISCUSSION TITLE
Öll teikni vídeó Trausta - 2022/9/1

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#16] [vefsida-ummæli]
# Öll teikni vídeó Trausta - 2022/9/1

Trausti hefur sameinað öll vídeóin sem hann hefur teiknað gegnum tíðina í eitt stórt myndband.

https://www.biskupsreglan.com/verkefni/teikni-video-trausta

**Comment by MatthiasGudni** [comment]
Þetta var nú meira óhappið þarna í fyrsta myndbandinu 🤦‍♂️
